LICENSE-binary

```diff
@@ -205,53 +205,54 @@
 This project bundles some components that are also licensed under the Apache
 License Version 2.0:
 
-audience-annotations-0.5.0
+audience-annotations-0.13.0
 commons-cli-1.4
 commons-lang3-3.8.1
-jackson-annotations-2.13.4
-jackson-core-2.13.4
-jackson-databind-2.13.4.2
-jackson-dataformat-csv-2.13.4
-jackson-dataformat-yaml-2.13.4
-jackson-datatype-jdk8-2.13.4
-jackson-datatype-jsr310-2.13.4
-jackson-jaxrs-base-2.13.4
-jackson-jaxrs-json-provider-2.13.4
-jackson-module-jaxb-annotations-2.13.4
-jackson-module-scala_2.13-2.13.4
-jackson-module-scala_2.12-2.13.4
+jackson-annotations-2.13.5
+jackson-core-2.13.5
+jackson-databind-2.13.5
+jackson-dataformat-csv-2.13.5
+jackson-dataformat-yaml-2.13.5
+jackson-datatype-jdk8-2.13.5
+jackson-datatype-jsr310-2.13.5
+jackson-jaxrs-base-2.13.5
+jackson-jaxrs-json-provider-2.13.5
+jackson-module-jaxb-annotations-2.13.5
+jackson-module-scala_2.13-2.13.5
+jackson-module-scala_2.12-2.13.5
 jakarta.validation-api-2.0.2
-javassist-3.27.0-GA
-jetty-client-9.4.48.v20220622
-jetty-continuation-9.4.48.v20220622
-jetty-http-9.4.48.v20220622
-jetty-io-9.4.48.v20220622
-jetty-security-9.4.48.v20220622
-jetty-server-9.4.48.v20220622
-jetty-servlet-9.4.48.v20220622
-jetty-servlets-9.4.48.v20220622
-jetty-util-9.4.48.v20220622
-jetty-util-ajax-9.4.48.v20220622
-jersey-common-2.34
-jersey-server-2.34
+javassist-3.29.2-GA
+jetty-client-9.4.51.v20230217
+jetty-continuation-9.4.51.v20230217
+jetty-http-9.4.51.v20230217
+jetty-io-9.4.51.v20230217
+jetty-security-9.4.51.v20230217
+jetty-server-9.4.51.v20230217
+jetty-servlet-9.4.51.v20230217
+jetty-servlets-9.4.51.v20230217
+jetty-util-9.4.51.v20230217
+jetty-util-ajax-9.4.51.v20230217
+jersey-common-2.39.1
+jersey-server-2.39.1
 jose4j-0.9.3
 lz4-java-1.8.0
 maven-artifact-3.8.4
 metrics-core-4.1.12.1
 metrics-core-2.2.0
-netty-buffer-4.1.78.Final
-netty-codec-4.1.78.Final
-netty-common-4.1.78.Final
-netty-handler-4.1.78.Final
-netty-resolver-4.1.78.Final
-netty-transport-4.1.78.Final
-netty-transport-classes-epoll-4.1.78.Final
-netty-transport-native-epoll-4.1.78.Final
-netty-transport-native-unix-common-4.1.78.Final
+netty-buffer-4.1.92.Final
+netty-codec-4.1.92.Final
+netty-common-4.1.92.Final
+netty-handler-4.1.92.Final
+netty-resolver-4.1.92.Final
+netty-transport-4.1.92.Final
+netty-transport-classes-epoll-4.1.92.Final
+netty-transport-native-epoll-4.1.92.Final
+netty-transport-native-unix-common-4.1.92.Final
 plexus-utils-3.3.0
 reload4j-1.2.19
+reload4j-1.2.25
```

Comment:
duplicate dependency?

Reply:
Yes, this is a known issue: https://issues.apache.org/jira/browse/KAFKA-15015 .

Reply:
ah right. Thanks!